Bottom: 41b2d9ddc17fcd431384c93016a3ba8f151673e7
Top:    4cd4c4db77fad3ffd46d21dce29d0edb4ed8f592
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:51:40 +0000

Refresh of more-play-with-quad-edge-data-STEP

---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index 982ef7d..40ef822 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -12,7 +12,7 @@ typedef struct {
   appearance *appear;
 
   /* XXX: STEP specific - breaks encapsulation */
-  int plane_identifier;
+  int surface_identifier;
   int face_identifier;
   int face_bound_identifier;
 } face3d;
diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 54a2f11..ccccca5 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -426,8 +426,8 @@ object3d_export_to_step (object3d *object, const char *filename)
                    next_step_identifier + 3, next_step_identifier, next_step_identifier + 1, next_step_identifier + 2,
                    next_step_identifier + 4, next_step_identifier + 3, face->radius);
 
-          face->plane_orientation_reversed = true;
-          face->plane_identifier = next_step_identifier + 4;
+          face->surface_orientation_reversed = true;
+          face->surface_identifier = next_step_identifier + 4;
           next_step_identifier = next_step_identifier + 5;
         }
       else
@@ -469,8 +469,8 @@ object3d_export_to_step (object3d *object, const char *filename)
                    next_step_identifier + 3, next_step_identifier, next_step_identifier + 1, next_step_identifier + 2,
                    next_step_identifier + 4, next_step_identifier + 3);
 
-          face->plane_orientation_reversed = false;
-          face->plane_identifier = next_step_identifier + 4;
+          face->surface_orientation_reversed = false;
+          face->surface_identifier = next_step_identifier + 4;
           next_step_identifier = next_step_identifier + 5;
         }
     }
@@ -590,7 +590,7 @@ object3d_export_to_step (object3d *object, const char *filename)
           fprintf (f, "#%i, ", ((contour3d *)contour_iter->data)->face_bound_identifier);
         }
       fprintf (f, "#%i)", ((contour3d *)contour_iter->data)->face_bound_identifier);
-      fprintf (f, ", #%i, %s ) ;\n", face->plane_identifier, face->plane_orientation_reversed ? ".F." : ".T.");
+      fprintf (f, ", #%i, %s ) ;\n", face->surface_identifier, face->surface_orientation_reversed ? ".F." : ".T.");
       face->face_identifier = next_step_identifier;
       next_step_identifier = next_step_identifier + 1;
